packages/arm-appservice/lib/operations/appServiceCertificateOrders.ts

```diff
@@ -0,0 +1,1451 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as msRest from "ms-rest-js";
+import * as msRestAzure from "ms-rest-azure-js";
+import * as Models from "../models";
+import * as Mappers from "../models/appServiceCertificateOrdersMappers";
+import * as Parameters from "../models/parameters";
+import { WebSiteManagementClientContext } from "../webSiteManagementClientContext";
+
+/** Class representing a AppServiceCertificateOrders. */
+export class AppServiceCertificateOrders {
+  private readonly client: WebSiteManagementClientContext;
+
+  /**
+   * Create a AppServiceCertificateOrders.
+   * @param {WebSiteManagementClientContext} client Reference to the service client.
+   */
+  constructor(client: WebSiteManagementClientContext) {
+    this.client = client;
+  }
+
+  /**
+   * List all certificate orders in a subscription.
+   * @summary List all certificate orders in a subscription.
+   * @param [options] The optional parameters
+   * @returns Promise<Models.AppServiceCertificateOrdersListResponse>
+   */
+  list(options?: msRest.RequestOptionsBase): Promise<Models.AppServiceCertificateOrdersListResponse>;
+  /**
+   * @param callback The callback
+   */
+  list(callback: msRest.ServiceCallback<Models.AppServiceCertificateOrderCollection>): void;
+  /**
+   * @param options The optional parameters
+   * @param callback The callback
+   */
+  list(options: msRest.RequestOptionsBase, callback: msRest.ServiceCallback<Models.AppServiceCertificateOrderCollection>): void;
+  list(options?: msRest.RequestOptionsBase, callback?: msRest.ServiceCallback<Models.AppServiceCertificateOrderCollection>): Promise<Models.AppServiceCertificateOrdersListResponse> {
+    return this.client.sendOperationRequest(
+      {
+        options
+      },
+      listOperationSpec,
+      callback) as Promise<Models.AppServiceCertificateOrdersListResponse>;
+  }
+
+  /**
+   * Validate information for a certificate order.
+   * @summary Validate information for a certificate order.
```

Comment:
Is AutoRest giving the same values for .Description and .Summary? Or are these identical in Swagger?

Reply:
The values must be identical in the swagger. I only output the summary if it is provided.